pkg/reconcilermanager/controllers/reconciler_base.go

```diff
@@ -424,12 +424,14 @@ func (r *reconcilerBase) deployment(ctx context.Context, dRef client.ObjectKey)
 	return deployObj, nil
 }
 
-func mountConfigMapValuesFiles(templateSpec *corev1.PodSpec, c *corev1.Container, valuesFrom []v1beta1.ValuesFileSources) {
+// mountConfigMapValuesFiles mounts the helm values files from the referenced ConfigMaps as files in the helm-sync
+// container.
+func mountConfigMapValuesFiles(templateSpec *corev1.PodSpec, c *corev1.Container, valuesFileRefs []v1beta1.ValuesFileRefs) {
 	var valuesFiles []string
 
-	for i, vf := range valuesFrom {
-		fileName := reconcilermanager.HelmConfigMapRef
-		mountPath := filepath.Join("/etc/config", vf.Name, vf.ValuesFile)
+	for i, vf := range valuesFileRefs {
+		mountPath := filepath.Join("/etc/config", fmt.Sprintf("helm_values_filepath_%d", i))
```

Comment:
if you change FILE_PATH to FILEPATH, you'll have to change helm_values_filepath to helm_values_file_path here and in the tests too.

Reply:
done